lessons/Early-Math/identity-property/langs/en-us/pages/discover-identity.adoc

```diff
@@ -0,0 +1,86 @@
+= Discover the Identity Property
+
+++++
+<style>
+  table {grid-template-rows: 1fr 3fr !important;}
+  div.circleevalsexp .value,
+  div.circleevalsexp .studentBlockAnswerFilled { min-width:unset; }
+</style>
+++++
+
+@n Read each expression in words and translate it into a Circle of Evaluation.
+
+@vspace{1ex}
+
+[.FillVerticalSpace, cols="^.^6a,^.^6a,^.^6a",stripes="none"]
+|===
+
+| Find the sum of 12 and 4. | Find the sum of 12 and 4. Multiply it by 1. | Find the sum of 12 and 4. Add 0.
+
+| @showsoln{(coe '(+ 12 4))}
+| @showsoln{(coe '(* (+ 10 2) 1))}
+| @showsoln{(coe '(+ (+ 12 4) 0))}
+|===
+
+What do you notice? @fitb{}{@ifsoln{All three Circles evaluate to 16. We've multiplied the sum by one, and then we added zero.}}
+
+@fitb{}{@ifsoln{Both result in getting the same outcome.}}
+
+@fitb{}
+
+@fitb{}
+
+
+@vspace{1ex}
+
+These Circles represent the Identity Property!
+
+- The Identity Property of Multiplication tells us that the product of 1 and any number is that number.
+
+- The Identity Property of Addition tells us that the sum of a number and zero is that number.
+
```

Comment:
Could also be articulated as "tells us that adding 0 does not change the value."